### PR TITLE
Content tweaks

### PR DESCRIPTION
**Pull request by akeranen**

### CODE REVIEW

File: draft-keranen-t2trg-rest-iot.md [hunk 1]
```diff
@@ -527,7 +529,8 @@ The knowledge about these identifiers as well as matching implementations have t
 A client begins interacting with an application through a GET request on an entry point URI.
 The entry point URI is the only URI a client is expected to know before interacting with an application.
 From there, the client is expected to make all requests by following links and submitting forms that are provided in previous responses.
-The entry point IRI can be obtained, for example, by manual configuration or some discovery process (e.g., DNS-SD {{RFC6763}} or Resource Directory {{I-D.ietf-core-resource-directory}}).
+The entry point URI can be obtained, for example, by manual configuration or some discovery process (e.g., DNS-SD {{RFC6763}} or Resource Directory {{I-D.ietf-core-resource-directory}}).
+For Constrained RESTful environments "/.well-known/core" relative URI is defined as a default entry point for requesting the links hosted by a server {{RFC6690}}.
```
Comment (mkovatsc) [inline draft-keranen-t2trg-rest-iot.md]: In this case, your entry point URI is the address of the device hosting the /.well-known/core resource... You have to know or discovery this first...

File: draft-keranen-t2trg-rest-iot.md [hunk 2]
```diff
@@ -536,15 +539,15 @@ Also some interactions with a RESTful IoT system are straighforward to design;
 a classic example of reading a temperature from a thermometer device is almost always implemented as a GET request to a resource that represents the current value of the thermometer.
 However, certain interactions, for example data conversions or event handling, do not have as straighforward and well established ways to represent the logic with resources and REST methods.
 
-The following sections describe how common design problems such as different interactions can be modeled with RESTful and what are the benefits of different approaches.
+The following sections describe how common design problems such as different interactions can be modeled with RESTful design and what are the benefits of different approaches.
```
Comment (mkovatsc) [inline draft-keranen-t2trg-rest-iot.md]: ... can be modeled with REST and ...

Comment (mkovatsc) [reply draft-keranen-t2trg-rest-iot.md]: or "with a RESTful design and", but it is unnecessarily longer :)

Comment (akeranen) [reply draft-keranen-t2trg-rest-iot.md]: Fixed now

File: draft-keranen-t2trg-rest-iot.md (hunk 1 above)
Comment (akeranen) [reply draft-keranen-t2trg-rest-iot.md]: Clarified that knowledge of the address is needed for .w-k

**Pull request merged**